.github/workflows/platforms.yml

```diff
@@ -0,0 +1,47 @@
+name: Platform builds
+
+on:
+  schedule: [cron: '0 14 * * *']
+
+jobs:
+  build_linux_platforms:
+    runs-on: ubuntu-latest
+    name: ${{ matrix.conf.name }}
+    if: github.event_name != 'pull_request' || contains('dreamer,kcgen,ant-222', github.actor) == false
```

Comment:
(just a comment)

In here event will always be `true`, as this is on schedule job, but perhaps this is preparation for running these jobs on push/PR?

Reply:
Oh right! That was a copy-and-paste error from when I previously had it in our linux workflow. Feel free to knock that out :-)